API/src/main/java/de/brandwatch/minianalytics/api/controller/QueryController.java

```diff
@@ -0,0 +1,47 @@
+package de.brandwatch.minianalytics.api.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.brandwatch.minianalytics.api.service.QueryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+
+@org.springframework.web.bind.annotation.RestController
+public class QueryController {
+
+    private static final Logger logger = LoggerFactory.getLogger(QueryController.class);
+
+    @Autowired
+    private QueryService queryService;
```

Comment:
Now that I've said it in the other PR: I _always_ recommend constructor-based injection. That being said, not everyone does it, and this is not wrong, so I'll leave it up to you. I am not gonna mention it every time now.